ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -400,6 +420,22 @@ - (FlutterError* _Nullable)onListenWithArguments:(id _Nullable)arguments eventSi
     {
         self.documentLoadedEventSink = events;
     }
+    else if([arguments intValue] == documentErrorId)
```

Comment:
Would be cleaner if you stored the `intValue` in a local var at the top, and then implemented this as a switch statement.